### DISCUSSION TITLE
Payment model for services

### DISCUSSION THREAD

**Discussion by rastislavcore** [Dokploy/dokploy#244] [Ideas]
First this is great project!

I have one question, @Siumauricio do you have plan to create payment/subscription model for the clients? I will be happy to participate in that effort.

**Comment by Siumauricio** [comment]
Hi @rastislavcore, Yes it is in my plans to create a cloud version of dokploy I don't have an estimated time yet but it is sure that I will do it.


Send me a message to my discord and we can see what ideas to implement in a cloud version!

https://discord.com/invite/2tBnJ3jDJc

**Comment by Vanja-S** [comment]
Would the cloud version represent more of a managed dokploy service? Or would it just allow for custom payment options, i.e. stripe integration for automatic payment invoices to whomever you host for?

**Reply by Siumauricio** [reply]
In this case a managed dokploy service!

**Reply by rastislavcore** [reply]
BTW guys. I am working on the payment providers for the services + billing system. That Dokploy can act as Hosting provider. But it is quite lengthy process, because a lot of code needs to be changed and I am following the license rules. No ETA yet. I think this solution will exponationly raise the usecases for Dokploy.
